include/abb_librws/rws_client.h

```diff
@@ -278,6 +278,18 @@ class RWSClient : public POCOClient
     std::vector<SubscriptionResource> resources_;
   };
 
+  /**
+   * \brief An enumeration of controller coordinate frames.
+   */
+  enum class Coordinate
```

Comment:
```suggestion
  enum Coordinate
```

Even if scoped enums provied better portability and type safety I don't think it should be added in this PR.